packages/ra-core/src/form/useFormGroup.ts

```diff
@@ -109,11 +110,13 @@ export const useFormGroup = (name: string): FormGroupState => {
     );
 
     useEffect(() => {
+        if (!formGroups) return;
         // Whenever the group content changes (input are added or removed)
         // we must update its state
-        return formGroups.subscribe(name, () => {
+        const unsubscribe = formGroups.subscribe(name, () => {
             updateGroupState();
         });
+        return unsubscribe;
```

Comment:
I don't think we have an `unsubscribe` method. We should add one

Reply:
Yes we do:

https://github.com/marmelab/react-admin/blob/131619a8bfca889968daab748f1993e25c763d54/packages/ra-core/src/form/FormGroupsProvider.tsx#L34-L38